docs/control_center_docs/pulsing_module_getting_started.rst

```diff
@@ -0,0 +1,157 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _pulsing_module_start_guide:
+
+########################################################################
+Getting Started with Vertiq's Pulsing Firmware with IQ Control Center
+########################################################################
+
+Before completing the following getting started guide, please ensure that you have read and completed our :ref:`IQ Control Center 
+guide <control_center_start_guide>`. It walks you through Control Center installation, module configuration, and basic testing options available 
+to all Vertiq modules and firmware types. The following document is meant to provide additional configurations and testing options available only on Vertiq's pulsing firmware.
+
+.. note::
+    The following images are captured using IQ Control Center version 1.5.3 as connected to a Vertiq 23-06
+
+******************************************************
+Hardware Configuration for Pulsing Propellers
+******************************************************
+Vertiq's pulsing firmware is meant to operate specifically with our UPXX propellers. Please ensure that you have completed the following guides 
+in order to learn how to attach the propeller to your module, and to configure your module for proper flight with the propeller.
+
+* :ref:`up12_installation_tutorial`
+* :ref:`up12_initial_configuration`
+
+*********************************************************
+Important Parameters for Flight Controller Integration
+*********************************************************
+
+The first set of configurations covered here are those that **must** be configured properly in order to control your module with a flight controller. 
+These parameters affect how your module communicates with, and is controlled by a flight controller. 
+
+* **Mode**: this parameter is available through the General tab, and defines whether your module accepts throttle commands as a voltage, a velocity, or a PWM. 
+  To learn more about the Mode parameter see :ref:`throttle_mode`
+* **Motor Direction**: this parameter is available through the General tab, and defines the direction that your module takes to be positive, either clockwise or 
+  counter-clockwise. You can also configure either 2D or 3D configuration to allow for mapping negative throttle commands.
```

Comment:
FC 2D/3D Mode also needs to be configured for this to make sense. I'm guessing we always use 2D mode with the pulsing props, Luca can probably confirm or deny that, but its still something that you need to ensure you are interpreting things correctly

Reply:
I didn't mention it here because it does not show up as a configuration parameter through the control center, and so it doesn't make sense to mention in a doc about getting started with the control center.

Reply:
I was not aware it did not show up in the Control Center for pulsing, but I see that is the case now. Makes sense .